### PR TITLE
ci: add section on using docker-compose.override

### PR DESCRIPTION
**Pull request by jason-p-pickering**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,26 @@ environment variable
 
 DHIS2 should be available at `http://localhost:8080/dhis2`.
 
+
+##### Overriding default values
+
+You can create a local file called `docker-compose.override.yaml` and override values from the main `docker-compose.yaml`
```
Comment (Philip-Larsen-Donnelly) [inline README.md:135]:
```suggestion
You can create a local file called `docker-compose.override.yml` and override values from the main `docker-compose.yml`

```

Comment (jason-p-pickering) [reply README.md]: Thanks. Fixed.

File: README.md [hunk 2]
```diff
@@ -129,6 +129,26 @@ environment variable
 
 DHIS2 should be available at `http://localhost:8080/dhis2`.
 
+
+##### Overriding default values
+
+You can create a local file called `docker-compose.override.yml` and override values from the main `docker-compose.yml`
+file. As an example, you might want to use a different version of the Postgres database and run it on a different port. 
+More extensive documentation of this feature is available [here]https://docs.docker.com/compose/extends/). Using the override
```
Comment (teleivo) [inline README.md:137]: I think you are missing a `(` before `https`

File: README.md [hunk 3]
```diff
@@ -129,6 +129,26 @@ environment variable
...
+```yaml
+version: "3.8"
+
+services:   
+  db:
+    image: postgis/postgis:14-3.3-alpine
+    ports:
+      - 127.0.0.1:6432:5432
+```
+
```
Comment (teleivo) [inline README.md:149]: here as well, one newline between the sections is enough

File: README.md [hunk 4]
```diff
@@ -129,6 +129,26 @@ environment variable
 
 DHIS2 should be available at `http://localhost:8080/dhis2`.
 
+
```
Comment (teleivo) [inline README.md:132]: one newline between the sections is enough 😄

File: README.md (hunk 2 above)
Comment (jason-p-pickering) [reply README.md]: thanks. fixed

File: README.md (hunk 3 above)
Comment (jason-p-pickering) [reply README.md:148]: Thanks. Fixed.

**Review by teleivo — APPROVED** [review]

**Review by radnov — APPROVED** [review]

**Pull request merged**